frontend/components/vault/VaultLiquidationTransactionFlow.vue

```diff
@@ -16,8 +16,23 @@
         <VaultLiquidationTransactionTable v-if="!wasLiquidated" class="my-4" :vault-transaction="vaultTransaction" />
         <TextBlock class="mt-4">
             <template v-if="wasLiquidated">
-                This vault was liquidated <TimeTill :date="vaultTransaction.pastLiquidations[0].liquidationDate" /> in
-                the transaction <FormatAddress :value="vaultTransaction.pastLiquidations[0].transactionHash" />.
+                This vault was last liquidated
+                <TimeTill :date="vaultTransaction.pastLiquidations[0].liquidationDate" /> in the transaction
+                <FormatAddress :value="vaultTransaction.pastLiquidations[0].transactionHash" />.
+                <WithdrawDaiFromVat
```

Comment:
Does this panel have rounded corners at the moment? Otherwise, let's put it into a separate empty div, so it will have them